src/view/prsTreeDataProvider.ts

```diff
@@ -54,6 +54,11 @@ export class PullRequestsTreeDataProvider implements vscode.TreeDataProvider<Tre
 		return node ? this._onDidChangeTreeData.fire(node) : this._onDidChangeTreeData.fire();
 	}
 
+	reveal(): void {
```

Comment:
🤞 #750 introduced a similar logic and has cache for tree items in Pull Request Changes View.